changelog.d/ea-104.fixed

```diff
@@ -0,0 +1,12 @@
+Fixed Kotlin parse error.
+
+Previously, code like this would throw a parse error
+
+```
+fun f1(context : Context) {
+    Foo(context).elem = var1
```

Comment:
yes the kotlin grammar used to be very bad and incomplete regarding assignments
and what can be on the lhs.
I hope the more recent versions are better.